contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -41,8 +28,7 @@ script: |
 
   WRAP_DIR=$HOME/wrapped
   HOSTS="x86_64-linux-gnu arm-linux-gnueabihf aarch64-linux-gnu riscv64-linux-gnu"
-  CONFIGFLAGS="--enable-glibc-back-compat --enable-reduce-exports --disable-bench --disable-gui-tests"
-  FAKETIME_HOST_PROGS="gcc g++"
+  CONFIGFLAGS="--enable-determinism --enable-glibc-back-compat --enable-reduce-exports --disable-bench --disable-gui-tests"
```

Comment:
Passing `--enable-determinism` here isn't going to do anything, as that option currently only enables an additional linker argument when building for Windows.

Reply:
:facepalm: indeed https://github.com/bitcoin/bitcoin/blob/a63a26f042134fa80356860c109edb25ac567552/configure.ac#L807-L811

Reply:
Why do I observe deterministic builds without faketime wrappers?